examples/short.py

```diff
@@ -1,15 +1,20 @@
 """Example showing short flags in defopt.
 
-You can add alternative short flags to arguments by passing a
-dictionary to `defopt.run` which maps flag names to single letters.
+By default, short flags are generated for keyword arguments that do not
+share their initial with other keyword arguments.
+
+You can add alternative short flags to arguments by passing a dictionary
+to `defopt.run` which maps flag names to single letters.  You can
+prevent automatic addition of short flags by passing an empty dictionary
+as argument.
```

Comment:
These module docstrings are intended to be brief descriptions of what's happening in the example rather than full explanations of the features. Most of this probably belongs under the flags section in features.rst. (I have ended up duplicating a lot of documentation myself, which is not ideal.)